src/services/review/RepoCheckerService.ts

```diff
@@ -345,62 +356,59 @@ export default class RepoCheckerService {
               const anchorTags = dom.window.document.querySelectorAll("a")
               forEach(anchorTags, (tag) => {
                 const href = tag.getAttribute("href")
-                const text = tag.textContent || ""
-                if (!href) {
+                const text = tag.textContent
+
+                if (!text) {
+                  // while rendered in the dom, unlikely to be
+                  // noticed by end citizen.
+                  return
+                }
```

Comment:
honestly i think we should still report this, because it's more likely that this is an error than not. it might also be tab targetted/picked up by screen readers because those rely on the dom structure.

Reply:
resolving this to talk about this in a single thread